### PR TITLE
Algolia Detection Improved & Version Detection added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Improvement

**`Examples:`**

**for version Detection using:  __algolia.algoliasearch.version**:

- https://stripe.com/docs/api
- https://api.jquery.com/
- https://classic.yarnpkg.com/lang/en/

**using a[href]:**

- https://fontawesome.com/search

### CODE REVIEW

File: src/technologies/a.json [hunk 1]
```diff
@@ -2253,6 +2253,7 @@
       "_ALGOLIA": ""
     },
     "description": "Algolia offers a hosted web search product delivering real-time results.",
+    "dom": "a[href*='algolia\\.com']",
```
Comment (nurbek91) [inline src/technologies/a.json:2256]: Technology is defined without this trace
image

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!

File: src/technologies/a.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/a.json:2256]: @nurbek91 - when I am checking the FontAwesome Website, it's not able to detect the Algolia, it's able to get detected with `"dom"` trace only

Comment (nurbek91) [reply src/technologies/a.json:2256]: @dhavalveera Can you open this example several times and wait until the technology is displayed

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**